docs/gke.md

```diff
@@ -0,0 +1,73 @@
+# Google Kubernetes Engine (GKE) Deployment Guide
+
+Federation v2 can be deployed to and manage [GKE](https://cloud.google.com/kubernetes-engine/) clusters. Since
+Federation v2 requires Kubernetes v1.11 or greater, you must create
+[Alpha Clusters](https://cloud.google.com/kubernetes-engine/docs/concepts/alpha-clusters) and manually specify
+the `cluster-version`. Alpha clusters will no longer be needed once GKE supports Kubernetes `v1.11` or greater. You can
+use the `$ gcloud container get-server-config` command to view the GKE cluster versions available to you.
+The following example deploys two GKE alpha clusters named `cluster1` and `cluster2` using Kubernetes version
+`1.11.2-gke.4`.
+
+```bash
+$ gcloud container clusters create cluster1 --enable-kubernetes-alpha --cluster-version 1.11.2-gke.4 \
```

Comment:
Looks like Google has removed support for this recently as I now get:

```bash
ERROR: (gcloud.container.clusters.create) ResponseError: code=400, message=Master version "1.11.2-gke.4" is unsupported.
```

Reply:
I see the same thing. Just sent a msg to the gke channel to see what is going on.